web/app/view/SettingsMenuController.js

```diff
@@ -53,6 +53,7 @@ Ext.define('Traccar.view.SettingsMenuController', {
     onUserClick: function () {
         var dialog = Ext.create('Traccar.view.UserDialog');
         dialog.down('form').loadRecord(Traccar.app.getUser());
+        dialog.lookupReference('testMailButton').setHidden(false);
```

Comment:
If admin will edit other user and see this button, he may think that it sends mail to other user. But it sends mail only him. It may confuse.